### PR TITLE
Feature/sinf 83 rename resources

### PR DESCRIPTION
**Pull request by adrianmilne**
This renames the main resources to align more with TechOps standards, and adds Tags where applicable

I've left the TODO on the Tags on the EC2 ECS services for now - need to revisit this at some point

I think (as discussed with Tom) - there is more refactoring coming up - so we need to revisit names and tags as we get into that - so this is just to grab some of the low hanging fruit as we know we have a re-provisioning of DEV/INT coming up

### CODE REVIEW

File: terraform/modules/memcached/main.tf [hunk 1]
```diff
@@ -1,31 +1,48 @@
...
-  cluster_id           = "memcached-spree"
+  cluster_id           = "scale-eu2-ec-spree-memcached"
   engine               = "memcached"
   node_type            = "cache.t3.medium"
   num_cache_nodes      = 2
   parameter_group_name = "default.memcached1.5"
   security_group_ids   = var.security_group_memcached_ids
   subnet_group_name    = aws_elasticache_subnet_group.ec.name
+
+  tags = {
```
Comment (tommyb82) [inline terraform/modules/memcached/main.tf]: Could we set the repeating tags in a global map then use `merge` to combine with the `AppType` - save a bit of repetition / boilerplate? Something like:

```
tags = merge(module.globals.project_resource_tags, {AppType = "MEMCACHED"})
```

File: terraform/modules/configs/deploy-all/main.tf [hunk 2]
```diff
@@ -102,6 +102,10 @@ data "aws_ssm_parameter" "spree_db_password" {
   with_decryption = true
 }
 
+data "aws_ssm_parameter" "elasticsearch_url" {
+  name = "/bat/sbx1-elasticsearch-url"
```
Comment (tommyb82) [inline terraform/modules/configs/deploy-all/main.tf]: Hardcoded environment - replace with `${lower(var.environment)}`

**Review by tommyb82 — CHANGES_REQUESTED** [review]
Looking great - just need to update that ssm param name 👍

File: terraform/modules/memcached/main.tf (hunk 1 above)
Comment (adrianmilne) [reply terraform/modules/memcached/main.tf]: Very nice idea - got it working - just about to push a commit with it in 👍

Comment (adrianmilne) [reply terraform/modules/memcached/main.tf]: Pushed

File: terraform/modules/configs/deploy-all/main.tf (hunk 2 above)
Comment (adrianmilne) [reply terraform/modules/configs/deploy-all/main.tf]: Should be fixed now 👍

**Review by tommyb82 — APPROVED** [review]
Fantastic stuff 👍

**Pull request merged**